src/components/scrollManager.js

```diff
@@ -0,0 +1,246 @@
+define([], function () {
```

Comment:
is this a new file, or is it some 3rd-party library?

Reply:
It is new (self-written).